test/pkg/testconfig/testconfig.go

```diff
@@ -1092,3 +1324,19 @@ func discoverMode(ptpConfigClockUnderTest []*ptpv1.PtpConfig) {
 	}
 	GlobalConfig.DiscoveredClockUnderTestPod = pod
 }
+
+// Helper function to identify WPC NIC
+func GetWPCNICIface(nodeName string, solIfName string) (string, []string, bool) {
```

Comment:
I don't see any need of this function. You can just call ptphelper.GetListOfWPCEnabledInterfaces(nodeName) directly.